cli/studio.py

```diff
@@ -84,7 +84,7 @@ def _build_single_scenario(clean: bool, allow_offset_map: bool, scenario: str):
 
 
 def _build_single_scenario_proc(
-    clean: bool, allow_offset_map: bool, scenario: str, semaphore: Semaphore
+    clean: bool, allow_offset_map: bool, scenario: str, semaphore: Semaphore # pytype: disable=invalid-annotation
```

Comment:
I would assume Semaphore is causing this. It seems kind of curious.

Reply:
Yes it is

Reply:
`multiprocessing.Semaphore` is a `Callable` and thus the `invalid-annotation` error. The semaphore is of type `multiprocessing.synchronize.Semaphore`. 

Hence, either use the appropriate type 
```suggestion
from multiprocessing import Process, Semaphore, synchronize
def _build_single_scenario_proc(
    clean: bool, allow_offset_map: bool, scenario: str, semaphore: synchronize.Semaphore): 
```
or simply leave the type hint empty.
```suggestion
def _build_single_scenario_proc(
    clean: bool, allow_offset_map: bool, scenario: str, semaphore):
```